src/Elastic.Apm.StartupHook.Common/StartupHookLogger.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Licensed to Elasticsearch B.V under
+// one or more agreements.
+// Elasticsearch B.V licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information
+
+using System;
+using System.IO;
+
+namespace Elastic.Apm.StartupHook.Common
+{
+	/// <summary>
+	/// Logs startup hook process, useful for debugging purposes.
+	/// </summary>
+	public class StartupHookLogger
```

Comment:
If it's just the `StartupHookLogger` that's shared between `ElasticApmAgentStartupHook` and `Elastic.Apm.StartupHook.Loader`, a single linked file (or even copy/paste!) would be less involved than a shared project :)

Reply:
Ok, I changed this to share a file from one project and link it into the other.